_visualisation/core/focus-depth.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Focus and Depth
+position: 1.06
+---
+
+Normally, diagrams will focus on a given Entity and show all Parties with either ownership or control interests in that Entity, either directly or indirectly. However, you may wish to adopt a different focus or limit which parties or interests are shown.
+{: .lead}
+
+
+## Focus
+
+Your diagram's focus determines which Parties and interests are included.
+
+### Parties
+
+**Diagrams have either an Object or Subject Focus.** That is, they either:
+
+* Focus on selected Objects (at the ends of Chains), then trace backwards all of their Beneficiaries.
+* Focus on selected Subjects (at the starts of Chains), and trace forwards all the Objects they are interested in.
+
+#### Object Focus
```

Comment:
In line with my other comments on subject / object - this would need to update.

It could be a case of talking about 'Owner focus' vs. 'Subject focus'?  

Or 'Interested Party Focus' vs 'Entity focus'? 

As in, an entity focussed diagram starts from a specific entity, whereas an 'Interested Party Focus' diagram start from a specific interested party.

Reply:
I'd go for 'Interested Party Focus' or 'Subject Entity focus'.

Reply:
Looks like we went with Entity and Owner here, but that's in line with the agreed vocab, so I'm happy to resolve this.